crates/typst/src/export/pdf/mod.rs

```diff
@@ -161,8 +162,8 @@ fn write_catalog(ctx: &mut PdfContext) {
     }
 
     let creator = eco_format!("Typst {}", env!("CARGO_PKG_VERSION"));
-    info.creator(TextStr(&creator));
-    xmp.creator_tool(&creator);
+    info.creator(TextStr(creator.as_str()));
+    xmp.creator_tool(creator.as_str());
```

Comment:
why this change?

Reply:
Accidental from reverting other changes. Removed.